README.md

```diff
@@ -26,339 +26,213 @@
-that demonstrate how to use the libraries. You can also develop your
-own clients that directly communicate with Triton using [HTTP/REST or
-GRPC protocols](docs/inference_protocols.md). There is also a [C
-API](docs/inference_protocols.md) that allows Triton to be linked
-directly into your application.
-
-A [Triton backend](https://github.com/triton-inference-server/backend)
-is the implementation that executes a model. A backend can interface
-with a deep learning framework, like PyTorch, TensorFlow, TensorRT or
-ONNX Runtime; or it can interface with a data processing framework
-like [DALI](https://github.com/triton-inference-server/dali_backend);
-or you can extend Triton by [writing your own
-backend](https://github.com/triton-inference-server/backend) in either
-[C/C++](https://github.com/triton-inference-server/backend/blob/main/README.md#triton-backend-api)
-or
-[Python](https://github.com/triton-inference-server/python_backend).
-Triton also allows developers to write backends that can send multiple
-responses for a request or not send any responses for a request. Backends
-and models that operate in this way are referred to as [decoupled
-backends and models](docs/decoupled_models.md).
-
-A [Triton repository agent](docs/repository_agents.md) extends Triton
-with new functionality that operates when a model is loaded or
-unloaded. You can introduce your own code to perform authentication,
-decryption, conversion, or similar operations when a model is loaded.
-
-## Additional Resources
-
-* Additional documentation, presentations, and examples are located in the
-  [NVIDIA Developer Zone](https://developer.nvidia.com/nvidia-triton-inference-server).
-
-* Specific end-to-end examples for popular models, such as ResNet, BERT, and
-  DLRM are located in the [NVIDIA Deep Learning Examples page on
-  GitHub](https://github.com/NVIDIA/DeepLearningExamples).
+configuration](docs/model_configuration.md) for the model.  
+
+- Optimize your models setting [scheduling and batching](docs/architecture.md#models-and-schedulers)
+  parameters and [model instances](docs/model_configuration.md#instance-groups).
+- Use the [Model Analyzer tool](https://github.com/triton-inference-server/model_analyzer)
+  to help optimize your model configuration with profiling
+- [Add custom operations to Triton if needed by your model](docs/custom_operations.md)
+- Enable model pipelining with [Model Ensemble](docs/architecture.md#ensemble-models)
+  and [Business Logic Scripting (BLS)](https://github.com/triton-inference-server/python_backend#business-logic-scripting)
+- Learn how to [explicitly manage what models are available by loading and 
+  unloading models](docs/model_management.md)
+
+#### Configure and Use Triton Inference Server
+
+- Read the [Quick Start Guide](docs/quickstart.md) on to run Triton Inference Server
+  on both GPU and CPU
+- Triton supports multiple execution models, called 
```

Comment:
Is it accurate to refer  triton `backends` as  `execution models`?
I am not able to think of a more accurate word at the moment.

Reply:
Good catch, I meant "Execution Engines"

Reply:
I don't want to overload the term "model"